### DISCUSSION TITLE
Update Zigbee2MQTT on Windows

### DISCUSSION THREAD

**Discussion by tonsemail** [Koenkk/zigbee2mqtt#15587] [General]
Hi

I have currently Z2M v 1`.23.0, and want to update to the latest verzion
What are the exact steps to take to do this on windows?

My Guess is:
- Stop Z2M
- secure the current data folder
- remove complete Z2M folder
- Install newest version
- replace data folder

Is that OK?

**Comment by Devongub** [comment]
This method was successful in my case.
After unpacking, run the command `npm ci` in the destination folder to install all dependencies.
